src/services/hawker.ts

```diff
@@ -0,0 +1,74 @@
+import Axios from "axios";
+import { ENDPOINTS } from "../constants/endpoints";
+import API  from '../types/api';
+
+/**
+ * Get all hawkers from the backend (Public)
+ * 
+ * @method GET
+ * @access Public
+ */
+export const getAllHawkers = async () => {
+    return await Axios.get(ENDPOINTS.BASE + ENDPOINTS.PRODUCTS)
```

Comment:
Err.. shouldn't you use `ENDPOINTS.HAWKERS` or something? You are calling the wrong API this whole page